resources/ext/ezbake.conf

```diff
@@ -2,8 +2,8 @@ ezbake: {
   pe: {}
   foss: {
     redhat: { dependencies: ["puppet >= 3.7.3", "puppet < 5.0.0"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] },
-    debian: { dependencies: ["puppet (>= 3.7.3-1puppetlabs1)", "puppet (<< 5.0.0-1puppetlabs1)"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] }
+              postinst: ["PATH=/opt/puppetlabs/bin:/usr/bin:$PATH puppetdb ssl-setup"] },
```

Comment:
When we switch to ezbake 0.3.x, will the /usr/bin even be needed? I mean, won't we have to go back and remove this old path anyway later on, since /usr/bin will be superfluous? I can't see the compatibility we want to keep anyway, maybe I'm missing something :-P.